lib/checkFlags.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+/**
+ * @module checkFlags
+ * @desc A module to encapsulate bitmasking flags
+ */
+
+
+var Flag = function (bit, mode) {
+    this.bit = bit;
+    this.mode = mode;
+};
+
+var Action = function (label, flag, data) {
```

Comment:
Needs comments. Flag should probably be the first param. The other two are optional, functionally. The last is purely optional